rfc8446.xml

```diff
@@ -4558,6 +4563,11 @@ target="extensions"/>, with any extension not listed there marked as "-" to
 indicate that it is not used by TLS 1.3.</t>
 </list></t>
 
+<t>This document updates an entry in the TLS Certificate Types registry
+originally created in <xref target="RFC6091"></xref>.  The entry for value 1
+is changed to have name "OpenPGP_RESERVED", Recommended value 'N', and comment
+"Used in TLS versions prior to 1.3."</t>
```

Comment:
Nice catch.

Reply:
Since the registry page refers to this RFC, I figure we needto mention it (as opposed to shoving it into 8447).